Web_Development/FrontEnd/HTML_Text_Formatting.md

```diff
@@ -0,0 +1,28 @@
+# Documentaion
+# HTML Text Formatting
+
+## HTML contains several elements or text for defining a special meaning
+## For example: We can format it into bold by using the ```<b>``` tag
+ ```<b>The Text is bold</b>```
+   Result: <b>The Text is bold</b>
+   
+ ## Similarly we can use other examples
+  1.For strong text:<br>
+  ```<strong>The Text is Strong</strong>```<br>Result: <strong>The Text is Strong</strong>
+  <br><br>
+  2. For italic text:<br>
+  ```<i>The Text is Strong</i>```<br>Result: <i>The Text is italic</i>
+  <br><br>
+  3.For emphasised text:<br>
+  ```<em>The Text is emphasised</em>```<br>Result: <em>The Text is emphasised</em>
+  <br><br>
+  4.For small text:<br>
+  ```<small>The Text is small</small>```<br>Result: <small>The Text is small</small>
+  <br><br>
+  5.For Subscripted text:<br>
+  ```<strong>The Text is <sub>Subscripted</sub></strong>```<br>Result: <strong>The Text is <sub>Subscripted</sub></strong>
+  <br><br>
+ 5.For Superscripted text:<br>
+  ```<strong>The Text is <sup>Subscripted</sup></strong>```<br>Result: <strong>The Text is <sup>Subscripted</sup></strong>
+  <br><br>
+  ### There are many more text editors  like the above examples
```

Comment:
What do you mean by text editors here?
If you meant more typographic tags. You could change the word, as it misleads the audience.

Reply:
ya ya sorry for that!!